custom_components/lghorizon/translations/nl.json

```diff
@@ -4,15 +4,15 @@
     "step": {
       "user": {
         "title": "LG Horizon - Provider en gebruikersnaam",
-        "description":"Vul de gegevens van je Ziggo, Telenet, UPC, Magenta of Virgin account in.",
+        "description": "Vul de gegevens van je Ziggo, Telenet, UPC, Magenta of Virgin account in.",
         "data": {
           "country_code": "Provider",
           "username": "Gebruikersnaam"
         }
       },
-      "credentials":{
-        "title": "LG Horizon - Credentials",
-        "description":"Vul de credentias van je Ziggo, Telenet, UPC/Sunrise, Magenta of Virgin account in.",
+      "credentials": {
+        "title": "LG Horizon - Inloggegevens",
+        "description": "Vul de inloggegevens van je Ziggo, Telenet, UPC/Sunrise, Magenta of Virgin account in.",
         "data": {
           "password": "Wachtwoord",
           "identifier": "DTV identifier",
```

Comment:
Is the string `Refresh Token` shown to the user? Would that need a translation?

Reply:
Yes, but in Dutch i still prefer the term Refresh token, since it connects with the documentation and translating in Dutch doesn't make sense in this case :)